Winmake 64 - add 64-bit version of pwsafe to make files and build scripts

dhocker opened the pull request:
The majority of this pull request adds release support for the 64-bit version of pwsafe and its associated tools like ResText and ResPWSL. I also added a new "build" make target to Makefile.windows. This target builds 32 and 64 bit versions of pwsafe (runs msbuild against pwsafe-14.sln). 

Unfortunately, I was only able to do a limited amount of testing on these changes (particularly Makefile.windows). A number of the build targets depend on resources I do not have (like PGP keys and code signing certs). Also, I couldn't test the upload targets (to GitHub and SourceForge).

There is one more piece of slightly related work that I need to do, but it is outside the scope of this  pull request. The Update_POs.vbs script needs to be reworked so it is driven by the Translations.txt file (like Create_DLLs.vbs).

ronys commented:
Further to our correspondence, let me know if/when I should try this out. GitHub doesn't think there's a conflict with the current master branch.

dhocker commented:
I've fixed the merge conflicts (I merged the latest master branch and fixed
the conflicts), so it should be ready to go.

On Thu, Mar 16, 2017 at 11:09 AM, ronys <notifications@github.com> wrote:

> Further to our correspondence, let me know if/when I should try this out.
> GitHub doesn't think there's a conflict with the current master branch.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/pwsafe/pwsafe/pull/295#issuecomment-287107429>, or mute
> the thread
> <https://github.com/notifications/unsubscribe-auth/ABpZ7rK1mgHaRAs5GXm4QfjqAP9lg50Lks5rmV7NgaJpZM4MeFvf>
> .
>



-- 
[dh]

ronys commented:
Made some minor tweaks, but nice work - thanks!
